_sass/_faq.scss

```
@@ -190,6 +203,33 @@
+    cursor: pointer;
+    padding: 4px 6px;
+    border-radius: 3px;
+    transition: opacity 0.15s ease, color 0.15s ease;
+    flex-shrink: 0;
+
+    &:hover {
+      color: $theme-color2;
+    }
+
+    &:active {
+      color: darken($theme-color2, 10%);
```

Stylelint (17.2.0)

[error] Expected color.adjust($color, $lightness: -$amount) instead of darken($color, $amount) (scss/no-global-function-names)

(scss/no-global-function-names)